Formula/a/atuin.rb

```diff
@@ -18,7 +18,7 @@ class Atuin < Formula
   depends_on "rust" => :build
```

Comment:
A new feature we introduced requires protoc to be installed now to build

I think this should sort it:

```suggestion
  depends_on "rust" => :build
  depends_on "protobuf" => :build
```

Reply:
Thanks @ellie!